test/Microsoft.Azure.WebJobs.Host.EndToEndTests/AsyncChainEndToEndTests.cs

```diff
@@ -550,9 +550,19 @@ public static void DisabledJob([QueueTrigger(Queue1Name)] string message)
 
         public static async Task BlobToBlobAsync(
             [BlobTrigger(ContainerName + "/" + Blob1Name)] Stream inputStream,
+            string blobTrigger,
+            Uri uri,
+            IDictionary<string, string> metadata,
+            BlobProperties properties,
             [Blob(ContainerName + "/" + Blob2Name, FileAccess.Write)] Stream outputStream,
             CancellationToken token)
         {
+            Assert.True(uri.ToString().EndsWith(blobTrigger));
+            string parentId = metadata["AzureWebJobsParentId"];
```

Comment:
Interesting that we use this custom metadata collection for SDK properties like the parent function ID, which is how we do the Dashboard correlation capabilities.